src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/OpprettBehandlingValg.tsx

```diff
@@ -92,7 +93,11 @@ const OpprettBehandlingValg: React.FC<IProps> = ({
         : minimalFagsak.status !== FagsakStatus.LØPENDE && kanOppretteBehandling;
     const kanOppretteRevurdering = !minimalFagsak
         ? false
-        : minimalFagsak.behandlinger.length > 0 && kanOppretteBehandling;
+        : minimalFagsak.behandlinger.filter(
+              behandling =>
+                  behandling.resultat !== BehandlingResultat.HENLAGT_FEILAKTIG_OPPRETTET &&
```

Comment:
Kunne kanskje vært en funksjon bare for å sørge for å hente alle henlagt saker, men ikke krise :)